pkg/providers/v1/aws.go

```diff
@@ -1668,7 +1668,16 @@ func (c *Cloud) NodeAddressesByProviderID(ctx context.Context, providerID string
 		if eni == nil || err != nil {
 			return nil, err
 		}
-		return getNodeAddressesForFargateNode(aws.StringValue(eni.PrivateDnsName), aws.StringValue(eni.PrivateIpAddress)), nil
+
+		// Assign NodeInternalIP based on IP family
+		if eni.Ipv6Addresses != nil {
+			internalIPv6Address := eni.Ipv6Addresses[0].Ipv6Address
+			klog.Infof("Processing IPV6 fargate node with instanceID: %s", string(instanceID))
```

Comment:
I don't think we want these on info level. Can imagine this would get quite noisy.

Reply:
Ack, removed the added log lines